x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/preview_pane.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiInMemoryTable, EuiInMemoryTableProps, SortDirection } from '@elastic/eui';
+import { getFlattenedFields } from '../../../../components/source_index_preview/common';
+import { ml } from '../../../../../services/ml_api_service';
+
+import { DataFramePreviewRequest } from '../../../../common';
+
+interface Transform {
```

Comment:
On latest `master` you should be able to use `DataFrameTransform` from `ml/public/data_frame/common/job.ts`.